src/js/components/ColumnCenter/doc.js

```diff
@@ -0,0 +1,27 @@
+import { describe, PropTypes } from 'react-desc';
+
+import { getAvailableAtBadge } from '../../utils/mixins';
+
+export const doc = ColumnCenter => {
+  const DocumentedColumnCenter = describe(ColumnCenter)
+    .availableAt(getAvailableAtBadge('ColumnCenter', 'Layout'))
+    .description('Centered column layout.')
+    .usage(
+      `import { ColumnCenter } from 'grommet';
+<ColumnCenter />`,
+    )
+    .intrinsicElement('div');
+
+  DocumentedColumnCenter.propTypes = {
+    gutter: PropTypes.oneOfType([
```

Comment:
I don't have an issue with `gutter`, but wondering if `gap` be more aligned with Grommet props? Understand though if we'd like to be more specific in this context.

Reply:
Good comment. I forgot to remove `ColumnCenter`, an older idea in this PR. I was looking more for `Columns`. I've removed ColumnCenter to reduce confusion. Sorry about that.